chpl/chpl-service/src/main/java/gov/healthit/chpl/datadog/OnDemandUrlCheckerManager.java

```diff
@@ -0,0 +1,203 @@
+package gov.healthit.chpl.datadog;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Component;
+
+import com.datadog.api.client.ApiException;
+import com.datadog.api.client.v1.model.SyntheticsAPITest;
+import com.datadog.api.client.v1.model.SyntheticsAPITestResultFull;
+import com.datadog.api.client.v1.model.SyntheticsApiTestFailureCode;
+import com.datadog.api.client.v1.model.SyntheticsGetAPITestLatestResultsResponse;
+import com.datadog.api.client.v1.model.SyntheticsTriggerBody;
+import com.datadog.api.client.v1.model.SyntheticsTriggerTest;
+
+import dev.failsafe.Failsafe;
+import dev.failsafe.RetryPolicy;
+import gov.healthit.chpl.exception.ValidationException;
+import gov.healthit.chpl.scheduler.job.urluptime.DatadogSyntheticsTestResultService;
+import gov.healthit.chpl.scheduler.job.urluptime.DatadogSyntheticsTestService;
+import gov.healthit.chpl.util.ErrorMessageUtil;
+import gov.healthit.chpl.util.Util;
```

Comment:
Calling out a few checkstyle alerts in this file. There is an Eclipse setting to auto-remove unused imports and also to remove trailing spaces. I believe they are "Save Actions"